src/components/ui-elements/BackButton.jsx

```diff
@@ -0,0 +1,16 @@
+import { Link } from "react-router-dom";
+
+const BackButton = () => {
```

Comment:
Make this component more generic.
The component is a link to `/`. Create a LinkButton component and pass the link using props